### DISCUSSION TITLE
NNI多目标搜索算法

### DISCUSSION THREAD

**Discussion by zl-hw** [microsoft/nni#3755] [Q&A] [answered]
## NNI 学生项目问题概述 / General Question of Student Program
您好，请问NNI有计划支持多目标的搜索算法吗？比如NSGA之类。谢谢

**Comment by QuanluZhang** [comment, marked as answer]
@zl-hw thanks for reporting this requirement. NNI as a framework can naturally support multi-objective optimization algorithm. It can been seen as a new tuner. We highly encourage external contributors to contribute such an algorithm to NNI.
